repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl

```diff
@@ -143,12 +143,18 @@
                 <div class="panel panel--grey">
                     <h3><@fmt.message key="headers.resources"/></h3>
                     <ul data-uipath="ps.publication.resources">
-                        <#list publication.attachments as attachment>
+                        <#list publication.attachmentsOld as attachment>
```

Comment:
Reference to this section needs to be added to the JIRA ticket referencing all bits of code that need removing once the data upgrade scripts are run (see comment in the review of `Publication.java`).

Reply:
Created RPS-238